build.gradle

```diff
@@ -40,9 +40,10 @@ dependencies {
 
 intellij {
     // this is the version of the IntelliJ IDEA API (SDK) used to build the plugin
-    version = '231.9011.34'
+    // https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#intellij-extension-version
+    version = '233.11799.30-EAP-SNAPSHOT'
```

Comment:
Does we have to wait until 2023.3 is out before merging this? Did we have some standard procedure on this?